### ISSUE TITLE
Android NPE on SDK 6.8.0

### ISSUE THREAD

**Issue by jarredt** [AppsFlyerSDK/appsflyer-react-native-plugin#412]
# Report

## Plugin Version

6.8.0

## On what Platform are you having the issue?

Android

## What did you do?

Published app including version 6.8.0 of SDK.

## What did you expect to happen?

Not see crash reports

## What happened instead?

See crash reports

## Please provide any other relevant information.

Stack trace:
```
java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String java.lang.String.replaceAll(java.lang.String, java.lang.String)' on a null object reference
        at com.appsflyer.internal.AFd1uSDK.values(unknown:4121)
        at com.appsflyer.internal.AFc1hSDK.AFInAppEventType(unknown:94)
        at com.appsflyer.internal.AFc1lSDK$4.run(unknown:109)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
        at java.lang.Thread.run(Thread.java:764)
```

 - So far, this appears to only be happening on devices running Android 7.1.2 or Android 9.
- The crash happens within a few seconds of our app starting (see attached breadcrumbs from Bugsnag).

![Screen Shot 2022-08-22 at 3 20 57 PM](https://user-images.githubusercontent.com/16692671/186029150-b403caa7-f483-4758-bc50-6077eff4715d.png)

![Screen Shot 2022-08-22 at 3 20 49 PM](https://user-images.githubusercontent.com/16692671/186029145-4d4fe396-75ac-4d81-b5a6-a55b8277ad5b.png)

**Comment by jarredt** [comment]
Note: we went from 6.6.1 to 6.8.0, so the issue may have been introduced in 6.7.0 or 6.8.0.

The stack trace makes me think it's related to this behavior introduced in 6.7.0: https://support.appsflyer.com/hc/en-us/articles/6971914107025-Bulletin-Adding-a-priority-queue-to-the-Android-SDK

**Comment by amit-kremer93** [comment]
possible duplicate of this:
 https://github.com/AppsFlyerSDK/appsflyer-android-app/issues/17

**Comment by jarredt** [comment]
@amit-kremer93 should I still email the support team?

**Comment by jarredt** [comment]
According to this, this issue was likely fixed in 6.8.1: https://github.com/AppsFlyerSDK/appsflyer-android-app/issues/17#issuecomment-1246033374. I'll close this for now while I update, but will reopen if we keep seeing in production.
